web-api/src/ingestion.rs

```diff
@@ -12,6 +12,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+mod preprocess_document;
```

Comment:
:art: we could just name it `preprocessing` or `preprocess` or sth similar. it's still a submodule of `ingestion` and won't cause name confusion, and it's more in line with the usual module naming.